src/main/java/io/nats/client/impl/NatsKeyValue.java

```diff
@@ -111,6 +112,7 @@ public KeyValueEntry get(String key, long revision) throws IOException, JetStrea
     }
 
     KeyValueEntry existingOnly(KeyValueEntry kve) {
+        Debug.info("_get", kve);
```

Comment:
Was this intended?

Reply:
Nope, which is why I don't commit the Debug class. I see now that the build failed.